icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/NumberFormatTest.java

```diff
@@ -72,6 +72,7 @@
 
     @Test
     public void TestRoundingScientific10542() {
+      assert false;
```

Comment:
Did you mean to include this change?

Reply:
Yes, it was intentional to see if Travis would fail, which it did.  I've reverted it now